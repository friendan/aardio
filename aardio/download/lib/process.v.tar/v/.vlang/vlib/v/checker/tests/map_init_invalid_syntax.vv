fn main() {
	a := map{}
	println(a)
}
