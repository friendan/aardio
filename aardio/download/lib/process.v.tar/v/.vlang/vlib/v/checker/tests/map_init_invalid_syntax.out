vlib/v/checker/tests/map_init_invalid_syntax.vv:2:10: error: invalid empty map initilization syntax, use e.g. map[string]int{} instead
    1 | fn main() {
    2 |     a := map{}
      |             ~~
    3 |     println(a)
    4 | }
